packages/core/core/src/Transformation.js

```diff
@@ -161,6 +162,16 @@ export default class Transformation {
           },
         ]);
       }
+
+      if (existing == null) {
+        // If no existing sourcemap was found, initialize sourcesContent with
+        // the asset's filepath and its original contents.
+        asset.sourcesContent = {
+          [path
+            .relative(this.options.projectRoot, asset.value.filePath)
+            .replace(new RegExp(path.sep, 'g'), '/')]: await asset.getCode(),
```

Comment:
Should this be https://github.com/parcel-bundler/parcel/blob/8b240d03dec1bec95bb232070de7cb60890171b1/packages/transformers/babel/src/babel7.js#L30 
instead?